ChiModules/LinearBoltzmannSolver/IterativeMethods/lbs_GMRES.cc

```diff
@@ -40,65 +44,53 @@ bool LinearBoltzmann::Solver::GMRES(LBSGroupset& groupset,
       << groupset.groups.back().id << "\n\n";
   }
 
+  //================================================== Get groupset dof sizes
+  size_t groupset_numgrps = groupset.groups.size();
+  auto num_delayed_ang_unknowns = groupset.angle_agg.GetNumberOfAngularUnknowns();
+  size_t local_size = local_node_count * num_moments * groupset_numgrps +
+                      num_delayed_ang_unknowns.first;
+  size_t globl_size = glob_node_count * num_moments * groupset_numgrps +
+                      num_delayed_ang_unknowns.second;
+
+  //================================================== Create PETSc vectors
+  phi_new = chi_math::PETScUtils::CreateVector(static_cast<int64_t>(local_size),
```

Comment:
Why are we casting these values to int64?

Reply:
Because that is what PETSc takes. On that note, we could create an issue to modify the `PETScUtils` so that those utility functions use `PetscInt` instead of `int64_t` (which would be more consistent). What do you think?

Reply:
Yes, I think that's probably the correct thing to do.  It sounds like it's at least the consistent thing to do.